netket/vqs/mc_expect_grad.py

```diff
@@ -380,3 +400,59 @@ def gradfun(der_loc_vals, der_logs_ave):
         LdagL_grad,
         model_state,
     )
+
+
+@partial(jax.jit, static_argnums=(0, 1))
+def _grad_expect_continuous(
```

Comment:
can't we recycle all of the content of this function from the discrete case? what is specific for the continuous case?
for example it seems to me that masses and other stuff is no longer needed?